functions/src/modules/SQR.functions.ts

```diff
@@ -101,41 +101,39 @@ export const sendEmailOnNewAllotment = functions.database.ref('/sqr/allotments/{
 //
 //      2. Remove DB entries for MP3s that don't exist (removeNonExistingMp3DBEntries)
 /////////////////////////////////////////////////
-
-exports.syncStorageToDB = functions.https.onRequest( async (req, res) => {
+export const syncStorageToDB = functions.https.onRequest( async (req, res) => {
     ///////////////////////////////////////////////////////
     //      1. Add the currently uploaded MP3s into the DB
     ///////////////////////////////////////////////////////
     const bucketFiles = await bucket.getFiles();
     bucketFiles.forEach(innerFilesObject => {
         innerFilesObject.forEach(file => {
-            helpers.storeFileNameToDB(file.name, db, 'sqr');
+            helpers.storeFileToDB(file.name, db, 'soundQualityReporting');
```

Comment:
As now files are stored in the root, you can make all these functions not SQR-specific. Indeed, once the files are uploaded to the storage, they should be added for both content reporting and sound quality reporting at once.